src/src/AppBundle/Controller/UnsubscribeController.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace AppBundle\Controller;
+
+use EmailBundle\Controller\EmailController;
+use AdminBundle\Entity\Registration;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+
+class UnsubscribeController extends EmailController
+{
+    /**
+     * @Route("/unsubscribe/{token}", name="unsubscribe_mailinglist")
+     */
+    public function indexAction($token)
+    {
+        $em = $this->getDoctrine()->getManager();
+        $registration = $em->getRepository(Registration::class)->findOneBy(array('confirmationToken' => $token));
+
+        if (null == $registration) {
+            throw $this->createNotFoundException('Invalid registration token.');
+        }
+
+        $attendee = $registration->getAttendee();
+        $language = $registration->getLanguages();
+        $eventType = $registration->getEvents()->getEventType();
+        $context = ['event_type' => $eventType];
```

Comment:
Registracia sa viaze aj na `event`, nie len `event_type`, takze aj ten treba pridat do contextu.

Reply:
ale ja to potom nikde inde uz nevyuzivam, cize neviem ci to ma zmysel to tam doplnit

Reply:
Ty to nevyuzivas, ale ktokolvek iny co bude pisat tie templaty to vyuzivat moze

Reply:
ok dobre